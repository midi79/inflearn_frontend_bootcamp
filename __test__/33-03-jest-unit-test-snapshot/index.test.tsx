import JestUnitTestPage from "../../pages/section33/33-02-jest-unit-test";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";

it("Test whether there are changing items or not ", () => {
  const result = render(<JestUnitTestPage />);
  expect(result.container).toMatchSnapshot();
});
