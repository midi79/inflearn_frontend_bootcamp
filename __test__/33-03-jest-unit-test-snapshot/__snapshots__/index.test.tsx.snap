// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test whether there are changing items or not  1`] = `
<div>
  <div>
    IU is 30 years old.
  </div>
  IU's favorite color : 
  <input
    type="text"
  />
  <button>
    Hang out with IU
  </button>
</div>
`;
